mlflow/models/model.py

```diff
@@ -497,7 +509,7 @@ def resources(self, value: Optional[Union[str, List[Resource]]]):
             serialized_resource = value
         self._resources = serialized_resource
 
-    def get_model_info(self):
+    def get_model_info(self, model_version=None):
```

Comment:
We also have `mlflow.models.get_model_info()`

```
info1 = mlflow.xxx.log_model(...)
info2 = mlflow.models.get_model_info(info1.model_uri)

# should info1 equal to info2?
```

Reply:
Good point! I think we'd better avoid modifying the `get_model_info` method here to avoid ambiguous, let me update PR.

Reply:
> Should info1 be equal to info2?

One tricky thing is that the `model_version` is not persisted in the yaml file, therefore, `info2` does not set the registered model version. I think this itself is fine, because persisting the version will cause another issue - what if user unregister it and register to different catalog. But to avoid confusion, shall we add a quick note in the docstring of `model_version` property?

Reply:
agreed, they don't have to be equal